lib/utils/analyze.js

```diff
@@ -7,20 +7,24 @@ function analyze(results, sorted = true) {
 		return output;
 	}
 
-	if (sorted) {
```

Comment:
For debugging purposes, sorting the results first seems like a potential footgun. Best to save it to the end, so that every run of the code generates the same intermediate steps.